horovod/spark/lightning/remote.py

```diff
@@ -52,6 +52,10 @@ def RemoteTrainer(estimator, metadata, ckpt_bytes, run_id, dataset_idx, train_ro
     terminate_on_nan = estimator.getTerminateOnNan()
     transformation_fn = estimator.getTransformationFn()
     transformation = transformation_fn if transformation_fn else None
+    edit_fields = estimator.getTransformationEditFields()
+    transformation_edit_fields = edit_fields if edit_fields else None
```

Comment:
why `transformation_edit_fields` is needed and we cannot use `estimator.getTransformationEditFields()` directly?